ios/browser/api/sync/brave_sync_api.mm

```diff
@@ -63,25 +112,32 @@
 
 // MARK: - TimeLimitedWords::ValidationStatus
 
-OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
-    BraveSyncAPIWordsValidationStatusValid = static_cast<NSInteger>(
+BraveSyncAPIWordsValidationStatus const BraveSyncAPIWordsValidationStatusValid =
+    static_cast<NSInteger>(
         brave_sync::TimeLimitedWords::ValidationStatus::kValid);
-OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
+BraveSyncAPIWordsValidationStatus const
     BraveSyncAPIWordsValidationStatusNotValidPureWords = static_cast<NSInteger>(
         brave_sync::TimeLimitedWords::ValidationStatus::kNotValidPureWords);
-OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
+BraveSyncAPIWordsValidationStatus const
     BraveSyncAPIWordsValidationStatusVersionDeprecated = static_cast<NSInteger>(
         brave_sync::TimeLimitedWords::ValidationStatus::kVersionDeprecated);
-OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
+BraveSyncAPIWordsValidationStatus const
     BraveSyncAPIWordsValidationStatusExpired = static_cast<NSInteger>(
         brave_sync::TimeLimitedWords::ValidationStatus::kExpired);
-OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
+BraveSyncAPIWordsValidationStatus const
     BraveSyncAPIWordsValidationStatusValidForTooLong = static_cast<NSInteger>(
         brave_sync::TimeLimitedWords::ValidationStatus::kValidForTooLong);
-OBJC_EXPORT BraveSyncAPIWordsValidationStatus const
+BraveSyncAPIWordsValidationStatus const
     BraveSyncAPIWordsValidationStatusWrongWordsNumber = static_cast<NSInteger>(
         brave_sync::TimeLimitedWords::ValidationStatus::kWrongWordsNumber);
 
+namespace brave_sync {
+const char kSyncAccountDeletedNoticePending[] =
+    "brave_sync_v2.account_deleted_notice_pending";
+const char kSyncFailedDecryptSeedNotice[] =
+    "brave_sync_v2.failed_decrypt_seed_notice_dismissed";
+}  // namespace brave_sync
+
```

Comment:
Lets remove this & sync pref observer stuff in general as its not needed

Reply:
Removed at https://github.com/brave/brave-core/pull/17369/commits/5f3ce95e34441b7c2257f28cde14f7b4f3ca5704